src/CPK.ts

```diff
@@ -302,6 +316,63 @@ class CPK {
     })
   }
 
+  async getEnabledSafeModules(): Promise<Address[]> {
+    if (!this.#contract) {
+      throw new Error('CPK contract uninitialized')
+    }
+    const modules = await this.#contract.call('getModules', [])
+    return modules
+  }
+
+  async isSafeModuleEnabled(moduleAddress: Address): Promise<boolean> {
+    if (!this.#contract) {
+      throw new Error('CPK contract uninitialized')
+    }
+    const modules = await this.#contract.call('getModules', [])
+    const selectedModules = modules.filter(
+      (mod: Address) => mod.toLowerCase() === moduleAddress.toLocaleLowerCase()
```

Comment:
An address is a string, so just `toLowerCase()` should be fine? I don't think there's a difference though.

Reply:
Good catch! It was the VSCode autocomplete. Thank you!